src/gurt/dlog.c

```diff
@@ -143,6 +147,43 @@ static const char *clog_pristr(int pri)
 	return norm[s];
 }
 
+/**
+ * d_log_tm_init: Initialize logging telemetry.
+ *
+ * \return	zero on success, nonzero on error.
+*/
+int
+d_log_tm_init()
+{
+	int i, rc = 0;
+	struct d_tm_node_t **nctrs;
+
+	clog_lock();
+
+	if (!d_log_xst.dlog_facs)
+		D_GOTO(unlock, -DER_UNINIT);
+
+	if (d_log_xst.fac_cnt == 0)
+		D_GOTO(unlock, 0);
+
+	nctrs = calloc(1, d_log_xst.fac_cnt * sizeof(*nctrs));
```

Comment:
This memory is presumably lost?  Probably not a problem in practice but NLT tests that no memory is lost.